recipes/amos/meta.yaml

```diff
@@ -44,3 +44,5 @@ requirements:
     - perl-dbi
     - jellyfish
     - python
+extra:
+  biotools: [amos]
```

Comment:
can we have here this syntax?
```
biotools:
   - amos
```

Reply:
was there a decision about the key?
```
tool_identifier:
  - ff
  - ggg
```

```
extras:
   biotools_id:
     - ff
   other_id:
     - gg
```

Reply:
@bgruening @mc-r the I Would like to have somethin that can be auto-resolve and dont need to much parsing: 

```
extras: 
   biotools:
       - amos
```
or 

```
extras: 
  identifiers:
      - biotools: amos 
```
The only problem with the second one is that we need to assume that every provider provides a unique identifier which in some cases probably is not 100% true. And this is one of the main reason we (@hmenager and myself) used this `[amos]`. This can help us in the future for example for one bioconda package to have all the pubmed ids like: `pubmed: [PMID:ZZZ, PMID:ZZZZ]`

Thanks a lot for this. Hope you guys love the idea @bioconda/core

Reply:
What about:

```
extras: 
  identifiers:
      - biotools: [amos]
```

Would that suit you @ypriverol and @bgruening ?

Reply:
+1 this was what we originally thought. Support multiple providers and multiple ids for same provider.

Reply:
Yes, why not. Maybe:
```
extra:
    ids:
        biotools: [amos]
```
What would be the advantage of adding a `-` before biotools?

Reply:
Hi @johanneskoester as a developer I will said 100% for the shorter 'ids' but as we should expect users doing annotation, probably 'identifiers' is more intuitive. Also because is not really long. But I can accept both. BTW, not real advantage, only that we can have also this:

'''
extra:
    identifiers:
         - biotools: [amos]
         - punned: [pmid:23456]
'''
What do you think?

Reply:
I hope we don't invent new prefix scheme like `punned` or `pmid:` -- where do we resolve those prefixes? Another prefix registry?

Use http://identifiers.org/  (which admitedly don't really do many tool registries today) or simply full URIs  - with clear recommendations/examples. That would then be automatically extensible without any further work.

Reply:
indeed @johanneskoester , your syntax is what i was thinking about, the dash is useless there i guess.

Reply:
@stain, this is a very interesting point, indeed. You mean, instead of introducing a biotools identifier, one would register a tool at identifiers.org, and use that identifier both in bioconda and biotools?

Reply:
Hi @stain agree with you the punned id was only an example to clarify that we can implement multiple providers. Agree with you about the identifiers.org idea. We can probably get in contact with Sarala or others in identifiers.org for supporting software. 

Something we need to agree in the scheme/implementation is if we want to support full URL or key value pairs.

Reply:
If you are happy with that I can trigger the discussion with identifier.org

Reply:
Are there cases where the biotools ID differs from the package name? Right now it seems redundant to list the name twice.

Reply:
Hi @tomkinsc the current annotations are the one we can "guess" from biotools + bioconda. The complete list with the mismatch is here https://github.com/BioContainers/biotools-bioconda-ids/blob/master/biotools-biocontainers-venn.ipynb we would like to be able to have a mechanism in bioconda to reference to other ids tools. This will enable to registries such as biocontainers.pro/registry/ to pull the metadata from metadata resources. The machanism we are proposing will also work for other identifiers: publications, identifier.org, etc.

Reply:
@ypriverol @hmenager wasn't the idea to link the entire URI(s)?
I thought the only discussion was if the URI is enough or if we want to use an additional annotation key?

Reply:
@bgruening We can use URL or the key-value pairs. The problems with the urls is that some of the resources can change their URL and break the system, key-value pair may work in the future better.

Reply:
Hi all, @stain: I have been talking to Sarala, the main developer/project lead from identifiers.org. The way they can encode in the future bio.tools would bee: 

```
identifiers.org\bio.tools\amos 
``` 
From my point of view, this should be a discussion trigger by bio.tools group with identifiers.org. If we have the current representation `- biotools: [amos]` we can resolve this. I need to say that I don't know any place that standardizes yet the ids of software, tools or packages. Then, it will take some time until something is out there stable. 

This PR is trying to move forward the standardization in bioconda of references to identifiers in other tools.

Reply:
FYI, there is already another source of software identifiers at identifiers.org: Research Resource IDentifier (RRID) https://identifiers.org/rrid/

Here's an example for Bowtie https://identifiers.org/rrid/RRID:SCR_005476

Reply:
Hi all: 
    I have seen Bio.tools definitions and discussing with @bgruening understanding better the 
    bio.tools id system. In this case, the identifier and URI is `http://bio.tools/amos` instead of 
    amos. Then the current proposal should be the following:   

```
extra:
   identifiers: 
       biotools: http://bio.tools/amos
```  
then the value of this key-value pair (the identifier) should be a URI.